src/modules/shared/pages/innovation/support/support-summary-list.component.ts

```diff
@@ -92,7 +95,7 @@ export class PageInnovationSupportSummaryListComponent extends CoreComponent imp
         this.sectionsList[2].unitsList = response.SUGGESTED.map(item => ({
           ...item, historyList: [], isLoading: false, isOpened: false,
           canDoProgressUpdates: false,
-          temporalDescription: item.support.start ? `Date: ${this.datePipe.transform(item.support.start, 'MMMM y')}` : ''
+          temporalDescription: item.support.start ? `Date suggested: ${this.datePipe.transform(item.support.start, 'MMMM y')}` : ''
```

Comment:
I don't think this is right. The BE is returning the date of the latest update of status, not the date of suggestion if I am not mistaken.